presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -263,7 +263,7 @@ public void testTopN()
         assertQuery("SELECT linenumber, NULL FROM lineitem ORDER BY 1 LIMIT 23");
     }
 
-    @Test
+    @Test (enabled = false)
```

Comment:
@vermapratyush These tests fails with
```
VeloxUserError:  NULLIF not supported natively
```

Disabling for now to keep the CI green. Shall I delete these tests?

Reply:
Yes, please go ahead.

Reply:
@amitkdutta @vermapratyush Would one of you create a GitHub issue to describe the problem? It would be nice to do that every time we disable a test and include a link to the issue in a comment next to disabled test.